docs/user-guide/walkthrough/k8s201.md

```diff
@@ -115,26 +122,29 @@ metadata:
   name: nginx-controller
 spec:
   replicas: 2
-  # selector identifies the set of Pods that this
+  # selector identifies the set of pods that this
   # replication controller is responsible for managing
   selector:
-    app: nginx
-  # podTemplate defines the 'cookie cutter' used for creating
+    name: nginx
+  # template defines the 'cookie cutter' used for creating
   # new pods when necessary
   template:
     metadata:
       labels:
         # Important: these labels need to match the selector above
         # The api server enforces this constraint.
-        app: nginx
+        name: nginx
     spec:
       containers:
-      - name: nginx
-        image: nginx
-        ports:
-        - containerPort: 80
+        - name: nginx
```

Comment:
might be a spaces vs tabs problem, I don't know if our yaml files are standardized.